### DISCUSSION TITLE
Script downloads more than what was requested

### DISCUSSION THREAD

**Discussion by exogenomic** [UltimaHoarder/UltimaScraper#948] [General]
First of all this project is freakin' amazing, thanks! I'm noticing some weirdness in the last few releases including the latest as of 3/19/21. I can auth successfully, select the Onlyfans site, default profile, then I get my list of subscriptions. If I type the # next to the subscription I want to download, it does its thing, but then goes off and starts randomly downloading stuff from other subscriptions instead of going back to the menu. Any thoughts on what would cause this?

**Comment by artyom2035** [comment]
Maybe you have scrape_paid_content set to true??? If that's the case, based on my observation, it'll scrape the # that you pick plus all accounts that you've purchased from in the past.
